src/Bicep.Core/ApiVersions/ApiVersion.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+
+namespace Bicep.Core.ApiVersions
```

Comment:
Could we move this under the linter namespace? Or something to indicate that this is ARM-specific?

Reply:
Moved to Bicep.Core.Analyzers.Linter: ba24bc356

Considered changing also to AzApiVersion, but there are multiple references to ApiVersion already, e.g. ResourceTypeReference.ApiVersion.  

Do you want me to do anything else?